renovate.json

```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": [
+		"config:base",
+		":rebaseStalePrs",
+		":enableVulnerabilityAlertsWithLabel(Security)",
+		":pinVersions",
+		":label(RenovateBot)",
+		":semanticCommits",
+		"schedule:weekly"
+	],
+	"npm": {
+		"stabilityDays": 3
+	},
+	"assigneesFromCodeOwners": true,
+	"assigneesSampleSize": 1,
+	"semanticCommitType": "build"
+}
```

Comment:
Explanation: 
1. will rebase the PR if it can automatically
2. Perform vulnerability scans on known packages using GitHub security - also add Security tag to the PR
3. Pin all versions
4. Add `RenovateBot` label (easy to filter out later )
5. Use semantic commits
6. Scan on a weekly basis (ie Monday mornings)
7. Before making PR on NPM deps, wait 3 days in case it gets unpublished
8. Automatically assign PR to someone matching the appropriate code owner as defined (separate PR will have that list)
9. Randomly select 1 person from that list
10. change the commit type to be build rather than chore

For #8, should that be reviewer instead?

Reply:
switched to reviewer, makes more sense here

Reply:
switched to Tues as maybe will get it in time for it to trigger